tensornetwork/matrixproductstates/finite_mps.py

```diff
@@ -132,22 +132,6 @@ def canonicalize(self, normalize: Optional[bool] = True) -> np.number:
     self.position(len(self.nodes) - 1, normalize=False)
     return self.position(pos, normalize=normalize)
 
-  def check_canonical(self) -> Tensor:
```

Comment:
Why is this deleted?

Reply:
Perhaps a bit over-aggressive, but it's the exact same implementation as the method in the parent class (BaseMPS). Don't mind putting it back though.

Reply:
That's what I assumed, just making sure. Let keep it deleted.